yinsolidated/parser.py

```diff
@@ -98,24 +93,22 @@ def namespace(self):
     @property
     def module_name(self):
         ancestor_prefixes = self.xpath(
-            'ancestor-or-self::yin:*[@module-name]/@module-name',
-            namespaces=_NSMAP
+            "ancestor-or-self::yin:*[@module-name]/@module-name", namespaces=_NSMAP
         )
         try:
             return ancestor_prefixes[-1]
         except IndexError:
-            raise MissingModuleNameError(self)
+            raise _error.MissingModuleNameError(self)
 
     @property
     def prefix(self):
         ancestor_prefixes = self.xpath(
-            'ancestor-or-self::yin:*[@module-prefix]/@module-prefix',
-            namespaces=_NSMAP
+            "ancestor-or-self::yin:*[@module-prefix]/@module-prefix", namespaces=_NSMAP
         )
         try:
             return ancestor_prefixes[-1]
         except IndexError:
-            raise MissingPrefixError(self)
+            raise _error.MissingPrefixError(self)
```

Comment:
Shouldn't the `error` module be public if these errors can be raised to external consumers of the API?

Reply:
it's re-exported from `yinsolidated`